### PR TITLE
Update and Consolidate Install docs Ubuntu 20.04 and CentOS 8

### PR DESCRIPTION
**Pull request by murrant**
To test:
```
pip3 install mkdocs mkdocs-material pymdown-extensions
mkdocs serve
```

image

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: doc/Installation/Install-LibreNMS.md [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+exit
+```
+
+# DB Server
+
+## Configure MySQL
+
+=== "Ubuntu 20.04 / Debian 10"
+    ```
+    vi /etc/mysql/mariadb.conf.d/50-server.cnf
```
Comment (Jellyfrog) [inline doc/Installation/Install-LibreNMS.md:88]: I would say it's better we stop writing what editor to use, or use $EDITOR

File: scripts/deploy-docs.sh [hunk 2]
```diff
@@ -9,21 +9,15 @@ fi
 
 pip3 install --upgrade pip
 pip3 install --user --requirement <(cat <<EOF
-Click==7.0
 future==0.18.2
```
Comment (Jellyfrog) [inline scripts/deploy-docs.sh:12]: All these were added as a poor man's lock file. Now the dependencies will change over time

File: doc/Installation/Install-LibreNMS.md (hunk 1 above)
Comment (murrant) [reply doc/Installation/Install-LibreNMS.md:88]: I plan to check that "edit" is available on all platforms.  That references the $EDITOR env var.

File: scripts/deploy-docs.sh (hunk 2 above)
Comment (murrant) [reply scripts/deploy-docs.sh:12]: @Jellyfrog these were dependencies of the old windmill theme.  (I'm 90% sure).   I did a clean env and installed the top level items and "locked them to that version".

File: doc/Installation/Install-LibreNMS.md (hunk 1 above)
Comment (murrant) [reply doc/Installation/Install-LibreNMS.md:117]: CentOS does not, open to suggestions.  I support vi or nano (which is more friendly to use)

Comment (Jellyfrog) [reply doc/Installation/Install-LibreNMS.md:117]: I suggest just writing "edit the file", ie. As a text and not a command. If you don't know how to edit a file, you're going to have a bad time anyway :(

Comment (murrant) [reply doc/Installation/Install-LibreNMS.md:117]: hmm, that removes the simplicity of just copy and pasting the box (with the nice button)

Comment (SourceDoctor) [reply doc/Installation/Install-LibreNMS.md:117]: why not simply "open `/etc/mysql/mariadb.conf.d/50-server.cnf`" and "add/update whatever"

Comment (murrant) [reply doc/Installation/Install-LibreNMS.md:117]: $ open /etc/mysql/mariadb.conf.d/50-server.cnf
bash: open: command not found

Comment (hanserasmus) [reply doc/Installation/Install-LibreNMS.md:117]: I agree with @murrant The nice thing about lnms installs have always been the copy and paste functionality. Especially for newcomers. I would keep the `vi` but that is just my 2c.

File: doc/Installation/Install-LibreNMS.md [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+    ```
+    dnf install policycoreutils-python-utils
+    ```
+    
+    ## Configure the contexts needed by LibreNMS:
+    
+    ```
+    semanage fcontext -a -t httpd_sys_content_t '/opt/librenms/html(/.*)?'
+    semanage fcontext -a -t httpd_sys_rw_content_t '/opt/librenms/(logs|rrd|storage)(/.*)?'
+    restorecon -RFvv /opt/librenms
```
Comment (TheGreatDoc) [inline doc/Installation/Install-LibreNMS.md:408]: CentOS 7 install also have `/opt/librenms/bootstrap/cache`. Is it not needed anymore to be allowed in selinux?

Comment (murrant) [reply doc/Installation/Install-LibreNMS.md:408]: the web server doesn't write to bootstrap/cache unless we run composer from it... which I don't think we will be doing.

Comment (murrant) [reply doc/Installation/Install-LibreNMS.md:408]: I'll drop it from the selinux :)

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-65-release-changelog-june-2020/12687/1

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-65-release-changelog-june-2020/12687/1

**Pull request merged**